jobsets/secp256k1.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv
+, autoconf
+, automake
+, libtool
+, secp256k1Src
+}:
+
+stdenv.mkDerivation {
+  name = "secp256k1";
+
+  src = secp256k1Src;
+
+  buildInputs = [ autoconf automake libtool ];
+
+  configurePhase = ''
+    ./autogen.sh
```

Comment:
adding `autoreconfHook` to the `nativeBuildInputs` will run autogen for you, and also bring autoconf and automake into scope for you
then the configure flags just need to be in `configureFlags`